newIDE/app/src/UI/DragAndDrop/DragAndDropContextProvider.js

```diff
@@ -1,8 +1,22 @@
 import { Component } from 'react';
-import MultiBackend from 'react-dnd-multi-backend';
-import HTML5toTouch from 'react-dnd-multi-backend/lib/HTML5toTouch';
+import HTML5Backend from 'react-dnd-html5-backend';
+import TouchBackend from 'react-dnd-touch-backend';
+import MultiBackend, { TouchTransition } from 'react-dnd-multi-backend';
 import { DragDropContext } from 'react-dnd';
 
+const HTML5toTouch = {
```

Comment:
Interesting, why this change? Should be tested in the mobile app, which had issue with the transition from what I remember

Reply:
It's to be able to define `delayTouchStart` parameter that defaults to 0.

Reply:
Coming back to this: would be worth a comment I think